nservicebus/azure-storage-queues/configuration.md

```diff
@@ -62,6 +62,13 @@ Defaults:
  * 32 in Version 7
 
 
+#### SerializeMessageWrapperWith
+
+Azure Storage Queues Transport wraps messages in a transport specific message containing message metadata. By default, Azure Storage Queues Transport uses the same serializer for the message wrapper as configured for the contained message. While NserviceBus Version 6 supports [multiple deserializers](/nservicebus/serialization#specifying-additional-deserializers) for messages, all endpoints in the same system needs to use the same serializer for the wrapper message. In NServiceBus Version 6 and above, it is possible to configure a different wrapper serializer by using the `SerializeMessageWrapperWith` option.
```

Comment:
>While NserviceBus Version 6 supports [multiple deserializers](/nservicebus/serialization#specifying-additional-deserializers) for messages, all endpoints in the same system needs to use the same serializer for the wrapper message. In NServiceBus Version 6 and above, it is possible to configure a different wrapper serializer by using the `SerializeMessageWrapperWith` option.

This doesn't read well. What version difference are we trying to highlight here? Is it version difference between NServiceBus Version 5 and 6?